architecture/core_concepts/pods_and_services.adoc

```diff
@@ -160,6 +161,57 @@ link:https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-gui
 has details about the functionality and purpose of pods.
 ====
 
+[[init-containers]]
+== Init Containers
+
+An link:https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[init container] is a container in a pod that is started before the pod app containers are started. Init containers can share volumes, perform network operations, and perform computations before the remaining containers start. Init containers can also block or delay the startup of application containers until some precondition is met. 
+
+When a pod starts, after the network and volumes are initialized, the init containers are started in order. Each init container must exit successfully before the next is invoked. If an init container fails to start (due to the runtime) or exits with failure, it is retried according to the pod 
+xref:../../dev_guide/configmaps.adoc#consuming-configmap-in-pods[`restartPolicy`]: 
+
+* `Always` tries restarting infinitely, with an exponential back-off delay of (10s, 20s, 40s) until the pod is restarted. 
+* `Never` does not try to restart. Pods immediately fail and exit.  
+* `OnFailure` tries restarting with an exponential back-off delay of (10s, 20s, 40s) capped at five minutes.
+
+A pod cannot be ready until all init containers have succeeded. 
+
+The following example outlines a simple pod which has two init containers. The first init container waits for `myservice` and the second waits for `mydb`. Once both containers complete, the Pod starts.
+
+.Sample Init Container Pod Object Definition (YAML)
+====
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp-pod
+  labels:
+    app: myapp
+spec:
+  containers:
+  - name: myapp-container 
+    image: busybox
+    command: ['sh', '-c', 'echo The app is running! && sleep 3600']
+  initContainers:
+  - name: init-myservice <1>
+    image: busybox
+    command: ['sh', '-c', 'until nslookup myservice; do echo waiting for myservice; sleep 2; done;']
+  - name: init-mydb <2>
+    image: busybox
+    command: ['sh', '-c', 'until nslookup mydb; do echo waiting for mydb; sleep 2; done;']
+----
+
+====
+
+<1> Specifies that the `myservice` container.
```

Comment:
Remove `that`

s/Specifies that the `myservice` container./Specifies the `myservice` container.